core/integration/module_config_test.go

```diff
@@ -242,6 +243,36 @@ func (ConfigSuite) TestConfigs(ctx context.Context, t *testctx.T) {
 			})
 		})
 	})
+
+	t.Run("Allows $schema keyword", func(ctx context.Context, t *testctx.T) {
+		c := connect(ctx, t)
+
+		baseWithOldConfig := c.Container().From(golangImage).
+			WithMountedFile(testCLIBinPath, daggerCliFile(t, c)).
+			WithWorkdir("/work").
+			With(daggerExec("init", "--source=.", "--name=test", "--sdk=go")).
+			WithNewFile("/work/main.go", `package main
+			type Test struct {}
+
+			func (m *Test) Fn() string { return "wowzas" }
+			`,
+			).
+			WithNewFile("/work/dagger.json", `{
+				"$schema": "https://docs.dagger.io/reference/dagger.schema.json",
```

Comment:
so this would end up like:

`https://docs.dagger.io/reference/schema/v0.14.0/dagger.schema.json`

Reply:
It's not currently versioned, so as written is correct.

Reply:
It *will* end up being correct though - see https://github.com/dagger/dagger/pull/9070.

Reply:
Ahh, yeah, makes sense!

Reply:
FWIW, if:

1. someone were to use 0.14.0
2. dagger releases some future version 0.15.0
3. user does _not_ update their local version to 0.15.0 (still on 0.14.0)
4. Their schema will reflect 0.15.0 which may include fields that are not supported by their local 0.14.0

If we wanted a versioned schema, best place would probably be from github itself: https://raw.githubusercontent.com/dagger/dagger/refs/tags/v0.14.0/docs/current_docs/reference/dagger.schema.json

But that also requires keeping this link up to date with whatever version you are using. _Personally_ that is more work than its worth for the benefit.